geomstats/geometry/special_euclidean.py

```diff
@@ -9,7 +9,9 @@
 from geomstats.geometry.euclidean import Euclidean
 from geomstats.geometry.general_linear import GeneralLinear
 from geomstats.geometry.invariant_metric import InvariantMetric
+from geomstats.geometry.lie_algebra import MatrixLieAlgebra
```

Comment:
Should we distinguish MatrixLieGroup and LieGroup then?

Reply:
I think LieGroup with the point_type argument works ok.